version/version.go

```diff
@@ -1,4 +1,47 @@
 package version
 
-// String is the version number of qri
-const String = "0.9.14-dev"
+import "fmt"
+
+var (
+	// Version is set by govvv at build time
+	Version = "n/a"
+	// GitCommit is set by govvv at build time
+	GitCommit = "n/a"
+	// GitBranch  is set by govvv at build time
+	GitBranch = "n/a"
+	// GitState  is set by govvv at build time
+	GitState = "n/a"
+	// GitSummary is set by govvv at build time
+	GitSummary = "n/a"
+	// BuildDate  is set by govvv at build time
+	BuildDate = "n/a"
+	// GolangVersion is set by govvv at build time
+	GolangVersion = "n/a"
+)
+
+// Map returns a summary of build info as a string map
+func Map() map[string]string {
+	return map[string]string{
+		"version":       Version,
+		"gitCommit":     GitCommit,
+		"gitBranch":     GitBranch,
+		"gitState":      GitState,
+		"gitSummary":    GitSummary,
+		"buildDate":     BuildDate,
+		"golangVersion": GolangVersion,
+	}
+}
+
+// Summary prints a summary of all build info.
+func Summary() string {
```

Comment:
Could we add a message here if the binary was built with `go install` by mistake? So, compare "Version" to "n/a", and if equal, return something like "please build using `make install` instead of `go install`".